routing_common/transit_types.hpp

```diff
@@ -23,6 +23,7 @@ using FeatureId = uint32_t;
 using OsmId = uint64_t;
 using Weight = double;
 using Anchor = uint8_t;
```

Comment:
Sort?

Reply:
I can't because `using Ranges = std::vector<std::vector<StopId>>;` needs `StopId`. But I sorted plan types. (Everything except form `using Ranges = std::vector<std::vector<StopId>>;`)